thavalon-server/src/lobby/client.rs

```diff
@@ -353,12 +359,40 @@ impl PlayerClient {
                             log::info!("Received new connection for client {}.", client_id);
                             to_client = ws;
                         }
+
+                        OutboundTaskMessageType::HeartBeat => {
```

Comment:
No need to change, but a possibly-simpler way to do this would be to have a `tokio::time::Interval` inside the `ToClient` task and then select on both `outbound_task_rx` and the `Interval` inside the loop. It's a different pattern than what's used elsewhere, but removes a task.